pandas/tests/groupby/test_timegrouper.py

```diff
@@ -408,7 +408,7 @@ def test_timegrouper_apply_return_type_series(self):
         df_dt["date"] = pd.to_datetime(df_dt["date"])
 
         def sumfunc_series(x):
-            return pd.Series([x["value"].sum()], ("sum",))
+            return pd.Series([x["value"].sum()], ("sum"))
```

Comment:
This one shouldn't change, as `('sum', )` is a tuple. The rest looks good though :+1: